flowz/test/artifacts/artifacts_test.py

```diff
@@ -9,142 +9,134 @@
 from flowz.artifacts import (ExtantArtifact, DerivedArtifact, ThreadedDerivedArtifact,
                              WrappedArtifact, TransformedArtifact, KeyedArtifact,
                              maybe_artifact)
-from flowz.channels import ChannelDone
+
+from ..channels.util import raises_channel_done
 
 
 class ArtifactsTest(tt.AsyncTestCase):
+    NAME = "Fooble"
```

Comment:
Uh, as a general convention, when I'm coming up with nonsense names, it's important that they start with "ph" and never a capital "F".  Get with the program.